content-src/components/Snippet/Snippet.scss

```diff
@@ -0,0 +1,77 @@
+.snippet {
+  $snippet-padding: 26px;
+  $snippet-bg-color: rgba($white, 0.95);
+  $snippet-divider-color: rgba($black, 0.1);
+  $snippet-width: 500px;
+  $snippet-base-font-size: 12px;
+  $snippet-title-size: 14px;
+  $snippet-image-size: 64px;
+  $snippet-image-margin: 15px;
+  $snippet-link-color: $link-blue;
+
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: $snippet-bg-color;
+  padding: $snippet-padding;
+  border-top: 1px solid $snippet-divider-color;
+  z-index: 1000;
+  align-items: center;
+
+  &.hidden {
+    transition: opacity 250ms ease-out, transform 250ms ease-out;
+    opacity: 0;
+    transform: translate3d(0, 100%, 0);
+  }
+
+  .snippet-container {
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    max-width: $snippet-width;
+    font-size: $snippet-base-font-size;
+  }
+
+  .snippet-image {
+    width: $snippet-image-size;
+    height: $snippet-image-size;
+    background-size: cover;
+    margin-right: $snippet-image-margin;
+    flex-shrink: 0;
+
+    &.placeholder {
+      border-radius: 8px;
+      border: 1px dashed $faintest-black;
+      background-color: $bg-grey;
+    }
+  }
+
+  .snippet-text {
+    a {
+      color: $snippet-link-color;
+      text-decoration: none;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    .snippet-title {
+      font-size: $snippet-title-size;
+      margin: 0 0 0.2em;
```

Comment:
Sounds reasonable; add a comment explaining this so noone inadvertently undoes it in the future?